### PR TITLE
Clarify Kops configuration

### PR DESCRIPTION
**Pull request by skopf**
- added hint to use EKS instead of Kops
- added note about configuring Kops after getting feedback

### CODE REVIEW

File: docs/helm-deployment-aws_cloud.md [hunk 1]
```diff
@@ -16,6 +18,11 @@ To run the Alfresco Content Services (ACS) deployment on AWS with Kubernetes req
 
 **Note:** You don't need to clone this repository to deploy Alfresco Content Services.
 
+### Prepare and configure Kops
+
+Kops requires certain configurations on your AWS account. Please follow the guide
+[here](https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-your-environment)
+through "Creating your first cluster". We will create the cluster for ACS in the next step.
```
Comment (killerboot) [inline docs/helm-deployment-aws_cloud.md]: Probably a better wording would be "The cluster for ACS will be created in the next step".

File: docs/helm-deployment-aws_cloud.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 # Alfresco Content Services Deployment with Helm on AWS
 
+**Hint:** Consider [deploying ACS into EKS](./helm-deployment-aws_eks.md) instead of setting
```
Comment (killerboot) [inline docs/helm-deployment-aws_cloud.md]: Probably better to add a note to the links in https://github.com/Alfresco/acs-deployment/blob/master/README.md#deployment-options that EKS is the recommended way for AWS deployments.

**Comment by skopf** [comment]
@anxumalo is already on the review list.

File: docs/helm-deployment-aws_cloud.md (hunk 2 above)
Comment (skopf) [reply docs/helm-deployment-aws_cloud.md]: 1. This note is for people who land here directly, e.g. by deep link from another project
2. Are we sure that EKS is now the recommended way?

**Pull request merged**